t/rose-ana/01-run-basic-v1.t

```diff
@@ -40,27 +39,27 @@ OUTPUT=$HOME/cylc-run/$NAME/log/job/1/rose_ana_t1/01/job.out
 TEST_KEY=$TEST_KEY_BASE-exact_numeric_success
 file_grep $TEST_KEY "[ OK ].*Semi-major Axis.*all: 0%:" $OUTPUT
 TEST_KEY=$TEST_KEY_BASE-exact_numeric_fail
-file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234%" $OUTPUT
+file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234000000000005%" $OUTPUT
```

Comment:
Probably not portable?

Reply:
Better truncated? i.e. `...5.234"`?

Reply:
I think it probably **is** portable... the default printing precision has just changed between Python 2 and 3... the alternative to updating the test would be to enforce a fixed formatting in `grepper.py` where the messages are printed, and I wanted to avoid doing that as that comes with its own problems when you can't be sure what values might arise there

Reply:
Left unchanged

Reply:
I'm pretty sure this isn't portable, the result could change with Python version / OS.

We don't have to change the way `rose_ana` presents results, just reduce the precision of the test to avoid breakage e.g:

```diff
-file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234000000000005%" $OUTPUT
+file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234*%" $OUTPUT
```

Reply:
`.*` rather than `*`. Would it even be safer to say `0*[0-9]`, since if it's a floating point change it should follow this pattern?

Reply:
`[0-9]*`